docs/howto/connection/gcp.rst

```diff
@@ -71,6 +71,12 @@ Scopes (comma separated)
         issue `AIRFLOW-2522
         <https://issues.apache.org/jira/browse/AIRFLOW-2522>`_.
 
+Number of Retries
+    Integer, number of times to retry with randomized
+    exponential backoff. If all retries fail, the HttpError
```

Comment:
Can you use the class role to create a link to the documentation
Example:
```
AAA :class:`~airflow.HttpError` BBBB
```
Reference: 
http://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html